rotkehlchen/chain/ethereum/graph.py

```diff
@@ -50,13 +61,8 @@ def get_common_params(
 class Graph():
 
     def __init__(self, url: str) -> None:
-        """
-        - May raise requests.RequestException if there is a problem connecting to the subgraph"""
```

Comment:
I've been looking at both `Client` and `RequestsHTTPTransport` init and I can't see a way they raising "requests-related" exceptions. All I've seen are `AssertionError` due to a misconfiguration during development.